.travis.yml

```diff
@@ -103,10 +108,13 @@ jobs:
 
     ## Deployment stage
     - <<: *deploy
-      # Deploy deb packages to bintray.
+      # Deploy deb packages to bintray. Doesn't build the packages but uses the
+      # cache from the first stage.
       stage: deploy
       if: tag IS present AND tag =~ ^v[0-9]+\.[0-9]+\.[0-9]+
       env:
         COMPILER_VERSION=4.9 MAKE_TARGET=$MAKE_TARGET:deb
-      after_success:
-        - beaver bintray upload -D $DIST_VERSION -N pkg/deb/*.deb
+      before_script: skip
+      script:
+        - beaver bintray upload -D $DIST_VERSION -N $CACHE_DIR/*.deb
+        - exit 0
```

Comment:
Why the `exit 0`?

Reply:
Wanting to finish the build as soon as possible. But see the point that it isn't required since build steps after `script`, `after_success` etc, have been defined. Will remove